Dockerfile

```diff
@@ -178,6 +178,11 @@ ENV NVIDIA_DRIVER_CAPABILITIES="compute,video,utility"
 
 ENV PATH="/usr/lib/btbn-ffmpeg/bin:/usr/local/go2rtc/bin:/usr/local/nginx/sbin:${PATH}"
 
+# Install mpp empowerred ffmpeg for arm64
```

Comment:
Is this going to run for all arm64 builds?

Reply:
Is there any reason this ffmpeg build wouldn't work for all arm64 use cases?  Could/Should we move it to be called from the install_deps.sh to be used instead of the raspbian repo?

Reply:
the rpi hwaccel has been very finicky https://github.com/blakeblackshear/frigate/issues/3780 so would definitely need testing on that

Reply:
in Docker, we have only one arm64, what "all" builds are you referring to?
Am I missing anything?

Reply:
> in Docker, we have only one arm64, what "all" builds are you referring to?

You are right

my question is basically that I don't see any path where HAS_FFMPEG isn't set to 1 and the mpp ffmpeg will always be used basically